Client/Frontend/Browser/DownloadToast.swift

```diff
@@ -9,14 +9,48 @@ class DownloadToast: Toast {
     struct UX {
         static let ToastBackgroundColor = UIColor.Photon.Blue40
```

Comment:
The background and progress color are not themeable?

Reply:
the background of the main view is set in the parent `Toast` and the progress is set on `applyTheme`